include/roboteam_world/net/netraw.h

```diff
@@ -0,0 +1,83 @@
+#ifndef _INCLUDED_NETRAW_H_
```

Comment:
This code is not ours and works, we could also just use QT's network capabilities though.